wiki/items/food.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Adding food
+index: 2
+---
+# Adding food
+This tutorial will assume that You have understood [Creating your first item](first-item)
+
+An item can be made food by adding a `FoodComponent` to the Item Settings which you can create by using the Builder. Try reading the javadocs of the builderto understand whats going on here or continue reading for an explanation:
+
+```java
+public static final Item EXAMPLE_FOOD = new Item(new QuiltItemSettings().food(new FoodComponent.Builder()
+    .hunger(2)
+	.saturationModifier(1)
+	.snack()
+    .alwaysEdible()
+    .meat()
+	.statusEffect(new StatusEffectInstance(StatusEffects.NIGHT_VISION, 10), 0.8f)
+    .build()));
+```
+Now that you tried using the javadocs *cough, cough*, here is an alternate explanation:
+- First, we instanciate a new builder. Using this, we can configure the `FoodComponent` before finally turning it into a proper `FoodComponent` using the build method. Note that all of the following values are by default 0, false or empty.
```

Comment:
```suggestion
1. we instantiate a new builder. Using this, we can configure the `FoodComponent` before finally turning it into a proper `FoodComponent`. Note that all of the following values are by default 0, false or empty.
```
use numbered list instead of 'first', 'then'...
moved `build` explanation to new final step to match actual order in code.